src/parser/TerraformParser.js

```diff
@@ -24,13 +24,45 @@ class TerraformParser extends DefaultParser {
    * Convert the content of files into Components.
    *
    * @param {FileInput[]} [inputs=[]] - Data you want to parse.
+   * @param {string} [parentEventId=null] - Parent event id.
```

Comment:
shouldn't it be `{String}`, like `{FileInput[]}` above ?

Reply:
no in jsdoc is string :)